assets/script.js

```diff
@@ -84,4 +84,78 @@ function getStreamingData(movieId) {
     });
 }
 
-// getStreamingData("tt0167260");
\ No newline at end of file
+// getStreamingData("tt0167260");
+
+
+// Gets movie data
+function getMovieData(movieId) {
+    var imdbApiUrl = "https://imdb-api.com/en/API/Title/k_zns86b2w/" + movieId;
+
+    fetch(imdbApiUrl).then(function (response) {
+        if (response.ok) {
+            response.json().then(function (data) {
+                console.log("--IMDb--");
+                console.log(data);
+                console.log(data.fullTitle);
+
+                let movieObject = {
+                    title: data.fullTitle,
+                    year: data.year,
+                    rating: data.imDbRating,
+                    image: data.image,
+                    plot: data.plot,
+                    streaming: getStreamingData(data.id)
+                };
+
+                console.log(movieObject);
+
+                return;
+            });
+        }
+    });
+}
+
+// add movie data to "movie-container"
+function addMovieData(movieObject) {
+    let movieContainer = document.querySelector("#movie-container");
+    let movieCard = document.createElement("div");
+    let movieImage = document.createElement("img");
+    let movieTitle = document.createElement("h2");
+    let movieYear = document.createElement("p");
+    let movieRating = document.createElement("p");
+    let moviePlot = document.createElement("p");
+    let movieStreaming = document.createElement("p");
+
+    movieCard.classList.add("movie-card");
+    movieImage.classList.add("movie-image");
```

Comment:
The image doesn't need a class, but the figure element will need a class of "image".